nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/action/TrashAction.java

```diff
@@ -86,7 +87,7 @@ public TrashComputation() {
 
         @Override
         protected void compute(CoreSession session, List<String> ids, Map<String, Serializable> properties) {
-            Boolean trashValue = (Boolean) properties.get(PARAM_NAME);
+            boolean trashValue = (boolean) properties.get(PARAM_NAME);
```

Comment:
In a case where `properties.get(PARAM_NAME)` returns null the code will fail at line `90` and  before this cleaning it failed on `if (trashValue)`. I think in a cleanup / format we not want to change the behaviour -> just keep the wrapper type.